packages/main-api/src/controllers/v1/spaces.rs

```diff
@@ -765,21 +913,28 @@ impl SpaceController {
         }
     }
 
+    
+
     pub async fn act_space_by_id(
         State(ctrl): State<SpaceController>,
         Extension(auth): Extension<Option<Authorization>>,
         Path(SpacePath { id }): Path<SpacePath>,
         Json(body): Json<SpaceByIdAction>,
-    ) -> Result<Json<Space>> {
+    ) -> Result<Response> {
```

Comment:
@Jamesunekwuojo 
Is there a reason why you changed the return type? 
If you change the return type, it seems like it will affect the update_space, posting_space API, etc. Did they work properly?